### PR TITLE
Fix some minor bug

### PR DESCRIPTION
**Pull request by muralx**
- Fix loss and rewards being swapped on plugins
- move all hyperparameters of dexp and trainable as constructor args
- shorter names of args to simplify command line
-

### CODE REVIEW

File: deep_quoridor/src/plugins/wandb_train.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def start_arena(self, game, total_games: int):
 
     def end_game(self, game, result):
         if self.episode_count % self.update_every == 0 or self.episode_count == (self.total_episodes - 1):
-            avg_reward, avg_loss = self.agent.compute_loss_and_reward(self.update_every)
+            avg_loss, avg_reward = self.agent.compute_loss_and_reward(self.update_every)
```
Comment (alejandromarcu) [inline deep_quoridor/src/plugins/wandb_train.py:36]: ooops, sorry 🤦

File: deep_quoridor/src/renderers/training_status.py [hunk 2]
```diff
@@ -19,15 +19,16 @@ def end_game(self, game, result):
...
                 print(
-                    f"{agent_name} Episode {self.episode_count + 1}/{self.total_episodes}, Steps: {self.step}, Avg Reward: {avg_reward:.2f}, "
+                    f"{agent_name} Episode {self.episode_count + 1}/{self.total_episodes} [{'*' if won else ' '}], Steps: {self.step}, Avg Reward: {avg_reward:.2f}, "
                     f"Avg Loss: {avg_loss:.4f}, Epsilon: {agent.epsilon:.4f}"
                 )
         self.episode_count += 1
         return
 
-    def action(self, game, step, agent, action):
+    def after_action(self, game, step, agent, action):
```
Comment (alejandromarcu) [inline deep_quoridor/src/renderers/training_status.py:31]: No action required, but in the future we should use the @override decorator so that there's an error if the base method is renamed

File: deep_quoridor/src/agents/core/trainable_agent.py [hunk 3]
```diff
@@ -203,6 +208,7 @@ def get_action(self, game):
         mask = observation["action_mask"]
 
         if random.random() < self.epsilon:
+            # print(f"rnd-mask: {mask}")
```
Comment (alejandromarcu) [inline deep_quoridor/src/agents/core/trainable_agent.py:211]: Maybe we should start using a log framework so that we can have lines like that and turn different levels of logging (we can do it later, no need to do anything now)

File: deep_quoridor/src/renderers/training_status.py [hunk 4]
```diff
@@ -19,15 +19,16 @@ def end_game(self, game, result):
         if self.episode_count % self.update_every == 0:
             for agent in self.agents:
                 agent_name = agent.name()
-                avg_reward, avg_loss = agent.compute_loss_and_reward(self.update_every)
+                avg_loss, avg_reward = agent.compute_loss_and_reward(self.update_every)
+                won = result.winner == agent.name()
                 print(
-                    f"{agent_name} Episode {self.episode_count + 1}/{self.total_episodes}, Steps: {self.step}, Avg Reward: {avg_reward:.2f}, "
+                    f"{agent_name} Episode {self.episode_count + 1}/{self.total_episodes} [{'*' if won else ' '}], Steps: {self.step}, Avg Reward: {avg_reward:.2f}, "
```
Comment (alejandromarcu) [inline deep_quoridor/src/renderers/training_status.py:25]: Nice addition

**Review by alejandromarcu — APPROVED** [review]
